### ISSUE TITLE
Doesn't know the answer 

### ISSUE THREAD

**Issue by mishav78** [facebookresearch/DrQA#273]
If the system doesn't know the answer will it return the best fitting paragraph?

**Comment by ajfisch** [comment]
It will always return whatever the argmax span is from the top-k paragraphs (even though they might be low confidence).

**Comment by mishav78** [comment]
will it also have an answer in addition to the top span?

**Comment by ajfisch** [comment]
The top span is the predicted answer.

**Comment by mishav78** [comment]
so will it have the paragraph it came from?

**Comment by ajfisch** [comment]
It's unclear what your question is. The model follows the same procedure for any input: its output will always consist of a single answer span. You can also see which paragraph and document the answer came from.

**Comment by mishav78** [comment]
so the bottom part is the paragraph? Is that correct?

Second question, can I get these components from within the program instead
of the shell?
for example in puesdocode
questioninstance.getSpan()
questioninstance.getParagraph()

>>> process('Who was the winning pitcher in the 1956 World Series?')

Top Predictions:
+------+------------+------------------+--------------+-----------+
| Rank |   Answer   |       Doc        | Answer Score | Doc Score |
+------+------------+------------------+--------------+-----------+
|  1   | Don Larsen | New York Yankees |  4.5059e+06  |   278.06  |
+------+------------+------------------+--------------+-----------+

Contexts:
[ Doc = New York Yankees ]
In 1954, the Yankees won over 100 games, but the Indians took the pennant with
an AL record 111 wins; 1954 was famously referred to as "The Year the Yankees
Lost the Pennant". In , the Dodgers finally beat the Yankees in the World
Series, after five previous Series losses to them, but the Yankees came back
strong the next year. On October 8, 1956, in Game Five of the 1956 World
Series against the Dodgers, pitcher Don Larsen threw the only perfect game in
World Series history, which remains the only perfect game in postseason play
and was the only no-hitter of any kind to be pitched in postseason play until
Roy Halladay pitched a no-hitter on October 6, 2010.

**Comment by ajfisch** [comment]
Yep, in that example the document is "New York Yankees" and the context is the paragraph from which "Don Larsen" (the answer) was extracted.

Predictions are returned as dictionaries, see the example code that generated the demo output: https://github.com/facebookresearch/DrQA/blob/main/scripts/pipeline/interactive.py#L79-L101

**Comment by mishav78** [comment]
so it means I would have to use the dictionary within the source code and
do something with the data as opposed to printing to the screen?
